aws/scylla_create_devices

```diff
@@ -23,16 +24,14 @@ import subprocess
 import urllib.request
 import urllib.error
 from pathlib import Path
-from lib.log import setup_logging
 
 
-setup_logging()
 logger = logging.getLogger(__name__)
 
 
 raid_script = "/opt/scylladb/scripts/scylla_raid_setup"
-raid_device = "/dev/md%d"
-scylla_root = "/var/lib/scylla"
+raid_device = ""
+scylla_root = ""
 
```

Comment:
Why are you changing code in the aws/ directory?

Reply:
If you think it is useful for non-aws, extract it to a library and use it from aws/k8s.

Reply:
Currently only AWS is supported. It's better to have scripts parameterized than relying on hardcoded values. These values doesn't work for k8s image so I added parametrization.

Reply:
that's fine, this script is per cloud, and we now have one for GCE as well...